components/atoms/Tooltip/tooltip.tsx

```diff
@@ -9,12 +9,13 @@ interface TooltipProps {
   delay?: number;
   className?: string;
   tipClassName?: string;
+  defaultOpen?: boolean;
 }
-const Tooltip = ({ children, content, className, direction }: TooltipProps): JSX.Element => {
+const Tooltip = ({ children, content, className, direction, defaultOpen }: TooltipProps): JSX.Element => {
   const { Portal, Root, Content, Trigger, Arrow } = TooltipPrimitive;
 
   return (
-    <Root delayDuration={300}>
+    <Root delayDuration={300} defaultOpen={defaultOpen}>
```

Comment:
`defaultOpen` starts with the tooltip open. We generally don't do that, but in the case of onboarding, there was a bespoke implementation of a tooltip with it always open. This  allows us to replace that bespoke tooltip. And when you click, it disappears and then functions like a tooltip normally does.